ravenframework/SupervisedLearning/MSR.py

```diff
@@ -552,9 +552,7 @@ def __evaluateLocal__(self,featureVals):
           #############
           ## OR
           #############
-          weights[key] = 0
-          for idx in indices:
-            weights[key] += self.__kernel(dists[:,idx]/h)
+          weights[key] = np.sum([self.__kernel(dists[:,idx]/h) for idx in indices])
```

Comment:
The return from ```self.__kernel(dists[:,idx]/h)``` can be float or numpy.array. Could you change your modification to ``` weights[key] = np.sum([self.__kernel(dists[:,idx]/h) for idx in indices], axis=0)```